src/test/java/com/ncc/aif/ScalingTest.java

```diff
@@ -259,62 +327,114 @@ private void addEventOrRelation() {
     }
 
     private void addEvent() {
-        // Add an event
+        // Add an event and add it to a cluster
         Resource eventResource = makeEvent(model, getEventUri(), system);
+        Resource eventCluster = makeClusterWithPrototype(model, getClusterUri(), eventResource, system);
+
+        if (introduceChaos(NIST_RESTRICTION.FlatClusters)) {
+            Resource entityCluster = makeClusterWithPrototype(model, getClusterUri(), getRandomEntity(), "handle", system);
+            markAsPossibleClusterMember(model, eventCluster, entityCluster, .5, system);
+        }
 
         // Set the type
-        Resource typeResource = SeedlingOntology.Business_DeclareBankruptcy;
+        Resource typeResource = LDCOntology.Conflict_Attack_AirstrikeMissileStrike;
         Resource typeAssertion = markType(model, getAssertionUri(), eventResource, typeResource, system, 1.0);
 
         addJustificationAndPrivateData(typeAssertion);
 
         // Make two arguments
         Resource argument = markAsArgument(model, eventResource,
-                SeedlingOntology.Business_DeclareBankruptcy_Place,
+                LDCOntology.Conflict_Attack_AirstrikeMissileStrike_Place,
```

Comment:
As we only have people, results, and numbers in our model, we might want to find events/relations that can have those things as arguments. It might make it easier to actually separate those things into their own collections so we can assign arguments correctly, but we don't have to do that now. The model doesn't really have to "make sense", but we should try to avoid violating the Ontology if we can.

Reply:
Yes-- I would love any suggestions on what ontology types we should use and where.  I tried to find something in LDC similar/analogous to the Seedling version that was already there, but once I got it to generate valid AIF, I stopped there.  But I'd love to improve it.

Reply:
LDCOwlOntology lists out types that each argument for relations/events can have. I just did a search for ldcOnt:Person and there are a couple (like Evaluate.Sentiment) where a person can be the "holder" and the "target".  Conflict.Attack can also have a person for "attacker" and "target". Maybe we just need to change which argument we're filling for the attack event?

Reply:
In the Seedling version of ScalingTest, Business_DeclareBankruptcy was being filled by Business_DeclareBankruptcy_Place and Business_DeclareBankruptcy_Organization (basically, a place and a "person."  I tried to do something analogous with Conflict_Attack_AirstrikeMissileStrike being filled by Conflict_Attack_AirstrikeMissileStrike_Place and Conflict_Attack_AirstrikeMissileStrike_Attacker-- also a place and a "person."  They were both under ldcOnt:Conflict.Attack and both were event argument types.  That was the extent of my investigation (and that the AIF was valid).  How should we change the attack event fillers?

Reply:
Made the changes we discussed on the phone so that things are more ontologically valid.